nilearn/masking.py

```diff
@@ -562,6 +566,52 @@ def compute_gray_matter_mask(target_img, threshold=.5,
     mask: nibabel.Nifti1Image
         The brain mask (3D image)
     """
+    return compute_brain_mask(target_img=target_img, threshold=threshold,
+                              connected=connected, opening=opening,
+                              memory=memory, verbose=verbose)
+
+
+def compute_brain_mask(target_img, threshold=.5, connected=True,
+                       opening=2, memory=None, verbose=0):
+    """Compute the whole-brain mask. This mask is calculated through the
+    resampling of the MNI152 template mask onto the target image.
+
+    Parameters
+    ----------
+    target_img: Niimg-like object
+        See http://nilearn.github.io/manipulating_images/input_output.html
+        Images used to compute the mask. 3D and 4D images are accepted.
+        Only the shape and affine of target_img will be used here.
+
+    threshold: float, optional
```

Comment:
```suggestion
    threshold : float, optional
```
The doc standards actually ask for a space between the attribute name and the colon (we did it wrong in the past)